src/main/java/org/sonar/plugins/stash/issue/StashDiffReport.java

```diff
@@ -46,7 +49,7 @@ public String getType(String path, long destination){
         break;
       } else{
         
-        if (StringUtils.equals(diff.getPath(), path) && (diff.getDestination() == destination)) {
+        if (StringUtils.equals(diff.getPath(), path) && ((diff.getDestination() == destination) || (includeVicinityIssues && Ranges.closed(diff.getSource() - PR_DIFF_COMMENT_OFFSET, diff.getDestination() + PR_DIFF_COMMENT_OFFSET).contains(destination)))) {

```

Comment:
This line is very long and complex. It would be better to externalize this check in a dedicated method in order to ease readibility.